File: app/src/main/java/com/bas080/autosleepdroid/PreferenceGetter.java
package com.bas080.autosleepdroid;

public interface PreferenceGetter {
    boolean getBoolean(String key, boolean defValue);
    int getInt(String key, int defValue);
    long getLong(String key, long defValue);
    String getString(String key, String defValue);
    boolean contains(String key);
}

File: app/src/main/java/com/bas080/autosleepdroid/PreferenceGetter.kt
package com.bas080.autosleepdroid

interface PreferenceGetter {
    fun getBoolean(key: String, defValue: Boolean): Boolean
    fun getInt(key: String, defValue: Int): Int
    fun getLong(key: String, defValue: Long): Long
    fun getString(key: String, defValue: String?): String?
    fun contains(key: String): Boolean
}
